internal/services/storage/storage_account_resource.go

```diff
@@ -239,6 +242,12 @@ func resourceStorageAccount() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"allow_shared_key_access": {
```

Comment:
could we change this to
```suggestion
			"shared_key_access_enabled": {
```
for consistency with the rest of the provider? (its how we typically name bools)